portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -44,10 +62,20 @@ public void doGet(HttpServletRequest request,
     Query query = new Query("Message").addSort("timestamp", 
                                                SortDirection.DESCENDING);
     PreparedQuery results = DATASTORE.prepare(query);
- 
+
+    // Get comment limit and convert to integer.
+    int commentLimit = 0;
```

Comment:
you should probably also check if commentLimit >= 0 at backend or provide default value such as
```commentLimit = max(0, commentLimit)``` . FetchOptions doesn't accept negative limits.
JS is easy to hack.